src/usrtpl.f

```diff
@@ -186,10 +187,12 @@ SUBROUTINE USRTPL(LUN,INVN,NBMP)
       CALL BORT(BORT_STR)
 904   WRITE(BORT_STR,'("BUFRLIB: USRTPL - REPLICATION FACTOR OVERFLOW'//
      . ' (EXCEEDS MAXIMUM OF",I6," (",A,")")') JVAL,TAG(NODI)
-      CALL BORT(BORT_STR)
+      call errwrt(bort_str)
```

Comment:
Is there a short/simple answer to why this returns a code as opposed calling bort as in the other cases? Is it to do with the context of the failure in operations (i.e. the calling programming needs to keep running)?

Reply:
Yes, as noted in #495, this particular stopped the operational GSI code dead in its tracks, so we needed to trap and move past that error without aborting the program.

As noted in https://github.com/NOAA-EMC/NCEPLIBS-bufr/discussions/340, we have a longer-term goal to refactor the library so that it never aborts an application program.  But that will be a longer-term task, so for now this PR can be viewed as a prototype for one way we might go about doing that.